docs/input-manifest/providers/cloudflare.md

```diff
@@ -1,16 +1,81 @@
 # Cloudflare
+Cloudflare provider requires `apiToken` token field in string format.
 
-For using your Cloudflare provider for DNS, you need to manually
+## DNS example
+```yaml
+providers:
+  cloudflare:
+    - name: cloudflare-1
+      apiToken: token
+```
 
-- [set up zone](https://developers.cloudflare.com/dns/zone-setups/)
-- Update your name servers for you domain with the NS records from the created DNS zone.
+## Create Cloudflare credentials
+You can create Cloudflare API token by following [this guide](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/). The required permissions for the zone you want to use are:
 
-since Claudie does not support their dynamic creation.
+```bash
+Zone:Read
+DNS:Read
+DNS:Edit
+```
 
-## IAM policies required by Claudie
+## DNS setup
+If you wish to use Cloudflare as your DNS provider where Claudie creates DNS records pointing to Claudie managed clusters, you will need to create a **public DNS zone** by following [this guide](https://developers.cloudflare.com/dns/zone-setups/).
 
-The token generated by Cloudflare for you zone has to have this policies
+!!! warning "Cloudflare is not my domain registrar"
+    If you haven't acquired a domain via Cloudflare and wish to utilize Cloudflare for hosting your zone, you can refer to [this guide](https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/#update-your-nameservers) on Cloudflare nameservers. However, if you prefer not to use the entire domain, an alternative option is to delegate a subdomain to Cloudflare.
 
-```sh
-Zone:Read, DNS:Read, DNS:Edit
-```
+## Input manifest examples
+
+### Load balancing example
+
+``` yaml
+name: CloudflareExampleManifest
+providers:
+  cloudflare:
+    - name: cloudflare-1 # Name of this provider instance.
+      apiToken: kslISA878a6etYAfXYcg5iYyrFGNlCxc # API token of this provider instance.
+    - name: aws-1
+      accessKey: SLDUTKSHFDMSJKDIALASSD
+      secretKey: iuhbOIJN+oin/olikDSadsnoiSVSDsacoinOUSHD
+
+nodePools: 
+  dynamic:
+    - name: loadbalancer
+      providerSpec:
+        name: aws-1
+        region: eu-central-1
+        zone: eu-central-1c
+      count: 2
+      serverType: t3.medium
+      image: ami-0965bd5ba4d59211c
+
+kubernetes:
+  clusters:
+    - name: cluster
+      version: v1.24.0
+      network: 192.168.2.0/24
+      pools:
+        control: []
```

Comment:
Just a heads up, applying this manifest will result in an error, maybe a comment saying that this manifest is to showcase Cloudflare and nothing else could be nice.

Reply:
added a warning callout stating that the example is not functional without cp and node pools